Jenkinsfile

```diff
@@ -3,6 +3,11 @@
 // Jenkins pipeline
 // See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
 
+// List of cloud targets
+def cloudTargetMatrix = [
+  "c4", "c5", "m4", "m5"
```

Comment:
why not also add `p2` `p3`?

Reply:
Let’s wait until the EC2 instance limit is increased?

Reply:
Make sense.